lib/readline.js

```diff
@@ -429,7 +428,9 @@ Interface.prototype._tabComplete = function() {
           }
         }
         handleGroup(self, group, width, maxColumns);
+      }
 
+      {
```

Comment:
@princejwesley @addaleax I think using `let` with the following [2 line](https://github.com/addaleax/node/blob/8c35e25ed106dfe7cbf9a7e171ac5aa097e4847c/lib/readline.js#L436-L437) could make the block start scoping there.